src/ut/httpstack_test.cpp

```diff
@@ -151,6 +158,18 @@ class HttpStackTest : public testing::Test
     return (int)rc;
   }
 
+  int post(const std::string& path,
```

Comment:
This is post with a body, yes? Can the name here be more descriptive, and can you comment what this function does?